datamodel/APC.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "APC",
+  "description": "Article Processing Charge status",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique identifier for the vocabulary",
+      "default": 1,
+      "type": "integer"
+    },
+    "vocabulary": {
+      "description": "The APC",
+      "type": "string",
+      "enum": ["paid", "partiallyWaived", "fullyWaived", "notCharged", "notRequired", "unknown"]
+    }
+  },
+  "required": ["id", "vocabulary"]
+}
```

Comment:
Suggest the JSON Schema files are tightened up through use of `"additionalProperties": false`, preventing additional properties from leaking into requests without being specifically added to the JSON Schema files.